src/styles/app/16-main-workspace-layout.css

```
@@ -0,0 +1,110 @@
+	flex-direction: column;
+	position: relative;
+	background: var(--bg-primary);
+	border-top-right-radius: 0;
+	border-bottom-right-radius: 0;
+	overflow: hidden;
+	transition: border-radius 180ms ease;
+}
+
+.canvasWrapper:has(.notesInfoSidebarHost:not(:empty)) .canvasPaneHost {
+	border-top-right-radius: calc(
+		var(--app-radius) -
```

Stylelint (17.9.0)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)